sys/vane/clay.hoon

```diff
@@ -2407,13 +2415,18 @@
     ++  allowed-by
       |=  {who/ship pax/path pes/regs}
       ^-  ?
-      =+  rul=rul:(read-p-in pax pes)
-      =-  ?:(?=($black mod.rul) !- -)
-      %-  ~(rep in who.rul)
-      |=  {w/whom h/_|}
-      ?:  h  &
-      ?:  ?=($& -.w)  =(p.w who)
-      (~(has in (fall (~(get by cez) p.w) ~)) who)
+      =/  rul=real  rul:(read-p-in pax pes)
+      =/  in-list=?  
+        ?|  (~(has in p.who.rul) who)
+          ::
+            %-  ~(rep by q.who.rul)
+            |=  {{@ta cru=crew} out/_|}
```

Comment:
Going by your changes you want to stick to old syntax. Friendly reminder that `=` isn't old syntax, `/` is. (Here, and every `=/` you used.)

(In case it was the previous usage with `=+` that threw you off, the `=` for `=+` is different ("give value a face") from the `=` for `=/` ("give face a type", sort of).)